Description: No need to start pulseaudio when starting lightdm user
 When starting the lightdm user, if pulseaudio is started,
 it will establish a relationship with bluetoothd. Afterwards,
 the system starts the pulseaudio of the user to be logged in,
 it also needs to establish a relationship with bluetoothd,
 the two pulseaudio conflict, so it needs to be closed
 lightdm user starts pulseaudio.
Author: zhaochengyi <zhaochengyi@uniontech.com>

Origin: https://gerrit.uniontech.com/plugins/gitiles/base/pulseaudio/+/ff25c4b85ed63c2676b42ba2a2d56cf0b4fe7931
Bug: https://pms.uniontech.com/zentao/bug-view-78593.html
Last-Update: 2022-05-25

--- pulseaudio-15.0+dfsg1.orig/src/daemon/systemd/user/pulseaudio.service.in
+++ pulseaudio-15.0+dfsg1/src/daemon/systemd/user/pulseaudio.service.in
@@ -15,6 +15,7 @@ Description=Sound Service
 # socket-service relationship, see systemd.socket(5).
 Requires=pulseaudio.socket
 ConditionUser=!root
+ConditionUser=!lightdm
 
 [Service]
 ExecStart=@PA_BINARY@ --daemonize=no --log-target=journal
--- pulseaudio-15.0+dfsg1.orig/src/daemon/systemd/user/pulseaudio.socket
+++ pulseaudio-15.0+dfsg1/src/daemon/systemd/user/pulseaudio.socket
@@ -1,6 +1,7 @@
 [Unit]
 Description=Sound System
 ConditionUser=!root
+ConditionUser=!lightdm
 
 [Socket]
 Priority=6
